src/main/java/data/Ui.java

```diff
@@ -0,0 +1,112 @@
+package data;
+
+import java.util.Scanner;
+
+/**
+ * Class that deals with receiving input from the user and sending output to the user.
+ */
+public class Ui {
+    /** Statement to show the user upon exit */
+    private static final String MESSAGE_WELCOME = "Hello! I'm Duke\n" + "What can I do for you?";
+
+    /** Statement to show the user upon running Duke */
+    private static final String MESSAGE_EXIT = "Bye. Hope to see you again soon!";
+
+    /** Statement to show the user upon finding no saved file */
+    private static final String MESSAGE_LOADING_ERROR = "No saved tasks were found.";
+
+    private static Scanner userInput = new Scanner(System.in);
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Shows message(s) to the user */
+    public void showToUser(String... message) {
+        for (String m : message) {
+            System.out.println(m + "\n");
+        }
+    }
```

Comment:
I think its really interesting how you created this showToUser function to print messages. Normally people would simply print the message, but you extracted it out and made the messages static constant variable